### ISSUE TITLE
💡 Jellyfin fork

### ISSUE THREAD

**Issue by djjudas21** [beluga-cloud/charts#318]
### How these charts can be improved?

Hi there. As you mentioned in https://github.com/beluga-cloud/charts/issues/302#issuecomment-2067638458 that the jellyfin chart is largely abandoned, I decided to fork it to [djjudas21/jellyfin](https://github.com/djjudas21/charts/tree/main/charts/jellyfin) which is also indexed on [Artifacthub](https://artifacthub.io/packages/helm/djjudas21/jellyfin).

I made three key changes:
* rework mediavolume support to work with new PVCs and existing PVCs
* switch to upstream [jellyfin/jellyfin](https://hub.docker.com/r/jellyfin/jellyfin) container image
* update to Jellyfin 10.9.x, which was required for compatibility with the app on my smart TV

Just wanted to let you know @xunleii that your code lives on, and for any users of `beluga-cloud/jellyfin` that an updated alternative now exists.

Thanks! :smile: 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by xunleii** [comment]
Thank you very much for your fork!

To be honest, I think the whole project will probably be stopped because I'm trying a different way of deploying things on my end 😅 .

I'll soon add a link to redirect to your chart on Artifact Hub in order. Have you forked another chart you maintain so I can communicate on it?

**Comment by djjudas21** [comment]
Yes, my fork is indexed on artifact hub at https://artifacthub.io/packages/helm/djjudas21/jellyfin

Happy for you to raise issues on my chart repo https://github.com/djjudas21/charts
